### PR TITLE
change default WaitServantJobTimeout && export param of WaitServantJo…

### PR DESCRIPTION
**Pull request by hhstu**
…bTimeout #567

Signed-off-by: 刘畅 <basefly@outlook.com>

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/yurtctl/cmd/convert/convert.go [hunk 1]
```diff
@@ -129,6 +129,8 @@ func NewConvertCmd() *cobra.Command {
 		"The yurthub image.")
 	cmd.Flags().Duration("yurthub-healthcheck-timeout", defaultYurthubHealthCheckTimeout,
 		"The timeout for yurthub health check.")
+	cmd.Flags().Duration("wait-servant-job-timeout", kubeutil.WaitServantJobTimeout,
+		"The timeout for servant-job run check.")
```
Comment (rambohe-ch) [inline pkg/yurtctl/cmd/convert/convert.go:133]: var wait-servant-job-timeout is unused?

Comment (hhstu) [reply pkg/yurtctl/cmd/convert/convert.go:133]: oh,sorry, fix it.

File: pkg/yurtctl/cmd/convert/convert.go [hunk 2]
```diff
@@ -216,6 +218,12 @@ func (co *ConvertOptions) Complete(flags *pflag.FlagSet) error {
 	}
 	co.YurthubHealthCheckTimeout = yurthubHealthCheckTimeout
 
+	waitServantJobTimeout, err := flags.GetDuration("wait-servant-job-timeout")
+	if err != nil {
+		return err
+	}
+	kubeutil.WaitServantJobTimeout = waitServantJobTimeout
```
Comment (rambohe-ch) [inline pkg/yurtctl/cmd/convert/convert.go:225]: I think we should not modify var `kubeutil.WaitServantJobTimeout`. how about add a `WaitServantJobTimeout` field for `ConvertOptions`?  and then use `co.WaitServantJobTimeout` for timeout of `RunServantJobs`.

Comment (hhstu) [reply pkg/yurtctl/cmd/convert/convert.go]: down

**Review by rambohe-ch — APPROVED** [review]

**Comment by rambohe-ch** [comment]
@hhstu please fix the golang lint error, and you can get error info in Details link.

**Pull request merged**